signer/tests/integration/postgres.rs

```diff
@@ -7142,7 +7142,7 @@ mod get_pending_accepted_withdrawal_requests {
     ///
     /// ```text
     ///
-    /// Height:       0           1           2     
```

Comment:
What tool you used to find this blankspaces? Maybe create an issue to add this tool to our CI?

Reply:
Oh, it is probably fmt/clippy but applied to all targets, maybe will be convered by Aldur's PR

Reply:
I searched manually with `[ \t]+\n` in vscode 😅

Reply:
WoW, nice!